src/components/Notifications/NotificationTypes/Referral/ReferralNotification.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { NotificationType } from "@dcl/schemas"
+import { ReferralCommonNotificationProps } from "./types"
+import { ReferralInvitedUserAcceptedIcon, RewardIcon } from "../../../Icon"
+import { NotificationItem } from "../../NotificationItem"
+import {
+  NotificationItemDescription,
+  NotificationItemTitle,
+} from "../../NotificationItem.styled"
+import { getBGColorByRarity } from "../../utils"
+
+const ReferralNotification = React.memo(
+  (props: ReferralCommonNotificationProps) => {
+    const { notification, locale, i18n } = props
+
+    const isNewTierReached =
+      notification.type === NotificationType.REFERRAL_NEW_TIER_REACHED
+
+    return (
+      <NotificationItem
+        image={
+          isNewTierReached ? (
+            notification.metadata.image
+          ) : (
+            <ReferralInvitedUserAcceptedIcon width={48} height={48} />
+          )
+        }
+        imageBackgroundColor={
+          isNewTierReached
```

Comment:
What do you think about avoiding the use of `!`? We had lots of problems in the past:
```suggestion
          isNewTierReached && notification.metadata.rarity
```